src/Session/commands.py

```diff
@@ -79,10 +79,44 @@ def list_freezers(self):
 
     def load(self, profile):
         pass
-    
-    def open(self, food_storage):
-        pass
+  
+    ''' Prompts user input to open a food storage. Also lists available food storages user can choose from. 
+        If a food storage is not available or a food storage with that name does not exist, prompts user to create a new one. '''
+    def open(self, foodStorage_type):
+        # Checks if a profile is loaded -> if not, prompts to load or create one
+        if not self.profile:
+            print(NO_LOADED_PROFILE_MESSAGE, \
+                SUGGESTED_ACTIONS_MESSAGE.format(listInQuotes([CREATE_PROFILE, LOAD])))
+            return
+
+        # Compiles list of food storages available
+        foodStorage_list = self.profile.listFoodStorages(foodStorage_type)
 
+        # If there are food storages available, prints open message and lists available food storages.
+        # If there are no food storages, prompts user to create one.
+        twoPrintOutcomes(foodStorage_list, [OPEN_FOOD_STORAGE_MESSAGE.format(foodStorage_type)], \
+            [NO_FOOD_STORAGE_MESSAGE.format(foodStorage_type)], [CREATE_FOOD_STORAGE, CREATE_FREEZER, CREATE_FRIDGE])
+
```

Comment:
One last thing is to return if foodStorage_list is empty after  `twoPrintOutcomes`